daemon/src/model/cfd.rs

```diff
@@ -39,12 +39,29 @@ impl<'v> FromParam<'v> for OrderId {
     }
 }
 
-#[derive(Debug, Clone, Serialize, Deserialize, PartialEq)]
+#[derive(Debug, Copy, Clone, Serialize, Deserialize, PartialEq)]
+/// Origin of the order
 pub enum Origin {
     Ours,
     Theirs,
 }
 
+/// Role in the Cfd
+#[derive(Debug, Copy, Clone, Serialize, Deserialize, PartialEq)]
+pub enum Role {
+    Maker,
+    Taker,
+}
+
+impl From<Origin> for Role {
```

Comment:
Perhaps add a comment that this assumption only holds until we only allow the maker to be the only one creating the offers?

Reply:
I think it always holds, because if you are in the role that created the offer, then you are always the maker.